packages/content-management/table_list/src/table_list_view.tsx

```diff
@@ -941,27 +995,28 @@ function TableListViewComp<T extends UserContentCommonSchema>({
         {/* Table of items */}
         <div data-test-subj={testSubjectState}>
           <Table<T>
+            addOrRemoveExcludeTagFilter={addOrRemoveExcludeTagFilter}
+            addOrRemoveIncludeTagFilter={addOrRemoveIncludeTagFilter}
+            clearTagSelection={clearTagSelection}
+            deleteItems={deleteItems}
             dispatch={dispatch}
-            items={items}
-            isFetchingItems={isFetchingItems}
-            searchQuery={searchQuery}
-            tableColumns={tableColumns}
+            entityName={entityName}
+            entityNamePlural={entityNamePlural}
             hasUpdatedAtMetadata={hasUpdatedAtMetadata}
-            tableSort={tableSort}
+            isFetchingItems={isFetchingItems}
+            items={items}
+            onSortChange={onSortChange}
+            onTableChange={onTableChange}
+            onTableSearchChange={onTableSearchChange}
             pagination={pagination}
+            searchQuery={searchQuery}
             selectedIds={selectedIds}
-            entityName={entityName}
-            entityNamePlural={entityNamePlural}
-            tagsToTableItemMap={tagsToTableItemMap}
-            deleteItems={deleteItems}
             tableCaption={tableListTitle}
+            tableColumns={tableColumns}
             tableItemsRowActions={tableItemsRowActions}
-            onTableChange={onTableChange}
-            onTableSearchChange={onTableSearchChange}
-            onSortChange={onSortChange}
-            addOrRemoveIncludeTagFilter={addOrRemoveIncludeTagFilter}
-            addOrRemoveExcludeTagFilter={addOrRemoveExcludeTagFilter}
-            clearTagSelection={clearTagSelection}
+            tableSort={tableSort}
+            tagsToTableItemMap={tagsToTableItemMap}
+            fixedTagReferences={fixedTagReferences}
```

Comment:
It seems that you wanted to sort the props by alphabetic order.. but added the new prop at the end 😊 Is that correct?  Is there any new prop to the component?

Reply:
IMO it would be easier to review if prop order were not change (here or in the Props interface)

Reply:
Sorry for the confusion, fixedTagReferences is the new prop.